projects/bunicorn/index.js

```diff
@@ -60,12 +114,13 @@ async function getTotalStableTVL(timestamp, ethBlock, chainBlocks) {
 }
 
 async function getTotalTVL(timestamp, ethBlock, chainBlocks) {
-  const [tokensSummary, stableSummary] = await Promise.all([
+  const [tokensSummary, stableSummary, farmSummary] = await Promise.all([
     getTotalTokenTVL(timestamp, ethBlock, chainBlocks),
-    getTotalStableTVL(timestamp, ethBlock, chainBlocks)
+    getTotalStableTVL(timestamp, ethBlock, chainBlocks),
+    getTotalFarmTVL(timestamp, ethBlock, chainBlocks)
```

Comment:
I'd prefer to move staking TVL into a staking function

Reply:
I fixed it